### DISCUSSION TITLE
validate_ucis_xml()

### DISCUSSION THREAD

**Discussion by nsdjg** [fvutils/pyucis#8] [General]
As is, this routine returns None when it returns successful.  It returns False when it fails.

Do you wish to maintain this or have it return True/False?  I am asking because I want to use this routine with a test of fc4sc.

**Comment by mballance** [comment]
Hmm... Interesting point. From my perspective, the exception is the most important thing in terms of signaling pass/fail. Looks like that will be thrown with the current code. Adding in a single 'ret = True' after the call to assertValid() shouldn't be a problem.
https://github.com/fvutils/pyucis/blob/f73a275179b2171b0e858426db55529a90a1b31b/src/ucis/xml/__init__.py#L45-L69
